Libraries/Components/FocusableWindows/FocusableWindows.windows.js

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Helper component for building keyboard focusable custom controls when the original native one has no such support
+ *
+ * The native implementation has special handling of this component with regard to layout/position: it mimicks the child's 
+ * position/size. So all properties (other than the keyboard/focus related ones) should be passed to the child.
```

Comment:
Instead of making this a comment, should we change FocusableControl such that it only supports the props we want?